src/network/networkbase.h

```diff
@@ -0,0 +1,166 @@
+#ifndef SERVERDATA_H
+#define SERVERDATA_H
+
+#include <QObject>
+#include <QString>
+#include <QDateTime>
+#include <QJsonArray>
+#include <QJsonObject>
+#include <QJsonDocument>
+#include <QUdpSocket>
+#include <QTcpServer>
+
+#define CUSTOM_DEBUG
+
+typedef quint16 PortNumType;
+typedef qint64 SizeType;
+const PortNumType g_defaultTcpPortNumber    = 36108;//36108
+const PortNumType g_defaultUdpPortNumber    = 32807;
+const SizeType    g_defaultBroadcastCycleMs = 500;
+const SizeType    g_defaultOutdatingCycleMs = 800;
+
+const QString     g_orgLabel             = "cpp-lv425";
+const QString     g_appLabel             = "PairStorm 0.0.1";
+
+struct ServerData {
+    QString               m_name;
```

Comment:
code conventional

Reply:
Refactoring is the latest step. Now it is repaired.